Find a functionally equivalent code:android.graphics.Path.addArc:COMMENT
Method Modifier: public      
Comment:/**
 * Add the specified arc to the path as a new contour.
 *
 * @param oval The bounds of oval defining the shape and size of the arc
 * @param startAngle Starting angle (in degrees) where the arc begins
 * @param sweepAngle Sweep angle (in degrees) measured clockwise
 */

@@ -1,7 +1,3 @@
 {
-    if (oval == null) {
-        throw new NullPointerException("need oval parameter");
-    }
-    isSimplePath = false;
-    native_addArc(mNativePath, oval, startAngle, sweepAngle);
+    addArc(oval.left, oval.top, oval.right, oval.bottom, startAngle, sweepAngle);
 }

